src/main/java/com/paypal/security/SecurityConfiguration.java

```diff
@@ -37,6 +37,15 @@
   private static final String SEC_PROPERTIES = "security.properties";
   private final Properties properties = new Properties();
 
+  private static final String NNA_PORT = "8080";
+  private static final String NNA_HISTORICAL = "false";
+  private static final String LDAP_ENABLED = "false";
+  private static final String AUTHORIZATION_ENABLED = "false";
+  private static final String LDAP_USE_STARTTLS = "false";
+  private static final String LDAP_CONNECT_TIMEOUT = "1000";
+  private static final String LDAP_RESPONSE_TIMEOUT = "1000";
+  private static final String NNA_SUGGESTIONS_RELOAD_TIMEOUT = "900000";
```

Comment:
I think we should rename the variables between lines 40-47. In Apache Hadoop we use the convention of suffixing the variables with _DEFAULT to signify that it is a default value. So I would expecting something like `NNA_PORT_DEFAULT` variable names.

Looks pretty good otherwise! Let's change the variable names and I'll do another check after that.

Reply:
Are we also looking to change the SecurityConfiguration class name as suggested in #47 or would that be for a separate issue?

Reply:
Nah, let's handle that as part of #10. :)

Reply:
Changed the variable names.